src/index.ts

```diff
@@ -8,3 +10,11 @@ export function orderRequest(): string {
 export function cancelOrder(id: number): string {
     return `Cancelled Order with ID: ${id}`;
 }
+
+export function quotationRequest(): Quotation {
+    let quotationPayload = QuotationPayloadBuilder.quotationPayload()
```

Comment:
use const if no further update to the var

Reply:
this is a dummy example for reference.